enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationOtpService.java

```diff
@@ -110,13 +132,13 @@ public void sendOtp(OwnerId ownerId, IdentityVerificationEntity identityVerifica
      * @throws OnboardingProcessException Thrown when onboarding process is not found.
      */
     public OtpVerifyResponse verifyOtpCode(String processId, String otpCode) throws OnboardingProcessException {
```

Comment:
Does it make sense to include `remainingAttempts` in the response from SCA point of view?

Reply:
Yes, this information is useful for the mobile application.